src/_P039_Thermosensors.ino

```diff
@@ -98,8 +99,14 @@
 # define P039_RTD_LM_TYPE            PCONFIG(6)
 # define P039_RTD_LM_SHTDWN          PCONFIG(7)
 # define P039_RTD_RES                PCONFIG_LONG(0)
+# define P039_FLAGS                  PCONFIG_ULONG(3)
+# define P039_TEMP_THRESHOLD_FLAG    0
 # define P039_RTD_OFFSET             PCONFIG_FLOAT(0)
+# define P039_TEMP_THRESHOLD         PCONFIG_FLOAT(1)
 
+# define P039_TEMP_THRESHOLD_DEFAULT (-273.15f) // Default and minimum value
```

Comment:
That's looking odd, with the float notation ;)

Reply:
That's supposed to be 0 K, as there won't be any lower temperatures 😉

Reply:
I know :)